src/components/GraphiQL/GraphiQL.tsx

```diff
@@ -0,0 +1,480 @@
+    "--font-size-h4": theme.typography.h4.fontSize,
+    "--font-size-hint": theme.typography.caption.fontSize,
+    "--font-size-inline-code": theme.typography.caption.fontSize,
+  } as React.CSSProperties;
+
+  const { theme: graphiqlTheme, setTheme: setGraphiqlTheme } = useGraphiqlTheme();
+  useEffect(() => {
+    if (theme.themeType !== graphiqlTheme) {
+      setGraphiqlTheme(theme.themeType)
+    }
+  })
+
+  return (
+    <div data-testid="graphiql-container" className="graphiql-container" style={rootStyle}>
+      <div className="graphiql-sidebar">
+        <div className="graphiql-sidebar-section">
+          {pluginContext?.plugins.map(plugin => {
+            const isVisible = plugin === pluginContext.visiblePlugin;
+            const label = `${isVisible ? 'Hide' : 'Show'} ${plugin.title}`;
+            const Icon = plugin.icon;
+            return (
+              <Tooltip key={plugin.title} label={label}>
+                <UnStyledButton
+                  type="button"
+                  className={isVisible ? 'active' : ''}
+                  onClick={() => {
+                    if (isVisible) {
+                      pluginContext.setVisiblePlugin(null);
+                      pluginResize.setHiddenElement('first');
+                    } else {
+                      pluginContext.setVisiblePlugin(plugin);
+                      pluginResize.setHiddenElement(null);
+                    }
+                  }}
+                  aria-label={label}
+                >
+                  <Icon aria-hidden="true" />
+                </UnStyledButton>
+              </Tooltip>
+            );
+          })}
+        </div>
+        <div className="graphiql-sidebar-section">
+          {/* <Tooltip label="Open short keys dialog">
+            <UnStyledButton
+              type="button"
+              onClick={() => setShowDialog('short-keys')}
+              aria-label="Open short keys dialog"
+            >
+              <KeyboardShortcutIcon aria-hidden="true" />
+            </UnStyledButton>
+          </Tooltip> */}
+        </div>
+      </div>
+      <div className="graphiql-main">
+        <div
+          ref={pluginResize.firstRef}
+          style={{
+            // Make sure the container shrinks when containing long
+            // non-breaking texts
+            minWidth: '200px',
+          }}
+        >
+          <div className="graphiql-plugin">
+            {PluginContent ? <PluginContent /> : null}
+          </div>
+        </div>
+        <div ref={pluginResize.dragBarRef}>
+          {pluginContext?.visiblePlugin ? (
+            <div className="graphiql-horizontal-drag-bar" />
+          ) : null}
+        </div>
+        <div ref={pluginResize.secondRef} style={{ minWidth: 0 }}>
+          <div className="graphiql-sessions">
+            <div
+              role="tabpanel"
+              id="graphiql-session"
+              className="graphiql-session"
+              style={{padding: '2rem 0 0 0'}}
```

Comment:
**suggestion**: inline styling - can we avoid that and use className if possible?